homework/style.css

```diff
@@ -1,3 +1,71 @@
 .alert-error {
```

Comment:
I guess you still have some styling work to do:

![image](https://user-images.githubusercontent.com/2788771/59553254-c7b47380-8f92-11e9-852f-b70ae30e2dbb.png)

![image](https://user-images.githubusercontent.com/2788771/59553260-db5fda00-8f92-11e9-8f62-9e7a2d8f5a76.png)

Reply:
Thanks! You are right! not totally complete, I'm working on it.